### DISCUSSION TITLE
Installation Error Docker Compose MariaDB" no matching manifest"

### DISCUSSION THREAD

**Discussion by seh2000** [alexjustesen/speedtest-tracker#733] [Q&A] [answered]
Hi,
New here and not a Linux guru.
I had the speedtest-tracker running for a few weeks on my Raspberry Pi, as now MariaDB is supported, I installed MariaDB and created a new DB, User, and Password.
Started a "new" installation with the new "script" with docker-compose up -d after a short while I got the below error message

Creating volume "speedtest-tracker_speedtest-db" with default driver
Pulling db (mariadb:10)...
10: Pulling from library/mariadb
ERROR: no matching manifest for linux/arm/v7 in the manifest list entries

Been searching the internet a bit, but didn't really figure out why/what to do.
Appreciate any help here!

**Comment by alexjustesen** [comment]
Best guess is the MariaDB image isn't available under the ARM architecture for that error message. Are you running this on an older raspberry pi or something else?

**Reply by seh2000** [reply]
Yes, I am running on a Raspberry Pi 3B+, I tried to remove mariadb I had installed manually and then re-run your script then I got same error  
 no matching manifest for linux/arm/v7 in the manifest list entries
There is apparetly a way to fix in manifest.tmpl but my knowledge about this is not enough.
I tried as well to run the scrip again after having re-installed mariadb and tried to define the env variables as alexdelprete suggested, but still got some error messages.

**Comment by alexdelprete** [comment]
> Pulling db (mariadb:10)...
> 10: Pulling from library/mariadb

Why are you pulling mariadb image if you already have mariadb installed separately?

If you have an existing mariadb instance, just define the env variables in the speedtest-tracker section of docker-compose and delete the mariadb section.

**Reply by seh2000** [reply]
Ahhh - I am a clown!
Completely overlooked that in the script, so I manually installed the mariadb 
Thanks Steen

**Comment by alexdelprete** [comment]
Please close the issue if issue is solved.

**Comment by seh2000** [comment, marked as answer]
All solved, just to share with others.
I had Raspberry OS 32bit running, re-flashed and installed Raspberry OS 64bit
Installed docker
Used Alex script under installation.
Now running 0.12-beta.1
Thanks to Alex and Alex
